plugins/woocommerce/client/blocks/packages/components/totals/item/index.tsx

```diff
@@ -61,7 +61,7 @@ const TotalsItem = ( {
 				{ label }
 			</span>
 			<DelayedContentWithSkeleton
-				isLoading={ ! value || showSkeleton }
```

Comment:
The visibility of the skeleton should only be dependent on the `showSkeleton` prop. 
The component can handle null, undefined, {}.